controllers/secret.go

```diff
@@ -24,187 +24,108 @@ import (
 	certv1alpha1 "github.com/redhat-openshift-ecosystem/operator-certification-operator/api/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 	k8errors "k8s.io/apimachinery/pkg/api/errors"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 const (
-	KUBECONFIG_SECRET = "kubeconfig"
-	GITHUB_API_SECRET = "github-api-token"
-	GITHUB_TOKEN      = "GITHUB_TOKEN"
-	PYXIS_API_SECRET  = "pyxis-api-secret"
-	PYXIS_API_KEY     = "pyxis_api_key"
+	KUBECONFIG_SECRET_NAME     = "kubeconfig"
+	KUBECONFIG_SECRET_KEY_NAME = "kubeconfig"
+	GITHUB_API_SECRET_NAME     = "github-api-token"
+	GITHUB_API_KEY_NAME        = "GITHUB_TOKEN"
+	PYXIS_API_SECRET_NAME      = "pyxis-api-secret"
+	PYXIS_API_KEY_NAME         = "pyxis_api_key"
 )
 
 // reconcileKubeConfigSecret will ensure that the kubeconfig Secret is present and up to date.
-func (r *OperatorPipelineReconciler) reconcileKubeConfigSecret(meta metav1.ObjectMeta) error {
-	pipelineName := types.NamespacedName{
-		Namespace: meta.Namespace,
-		Name:      meta.Name,
-	}
-	pipeline := &certv1alpha1.OperatorPipeline{}
-	err := r.Client.Get(context.TODO(), pipelineName, pipeline)
+func (r *OperatorPipelineReconciler) ensureKubeConfigSecret(meta metav1.ObjectMeta) error {
+	operatorPipeline, err := r.getPipeline(meta)
 	if err != nil {
-		if k8serrors.IsNotFound(err) {
-			log.Info("pipeline resource not found. Ignoring since object must be deleted")
-			return nil
-		}
-		log.Error(err, "unable to retrieve pipeline resource in namespace "+meta.Namespace)
+		log.Error(err, "unable to resolve kubeconfig secret for %s in %s", meta.Name, meta.Namespace)
```

Comment:
You could probably use namespace/name format.

Reply:
yeah would be nicer